corehq/motech/models.py

```diff
@@ -276,6 +277,12 @@ def soft_delete(self):
         self.is_deleted = True
         self.save()
 
+    def encrypted_fields(self):
```

Comment:
I'm not sold on the name "encrypted_fields" and am open to suggestions.

Reply:
Could this could be a class level property?

Reply:
I mainly did this to mimic how `natural_key` works with the django serializer. What are the benefits to making this a class level property vs a method?

Reply:
Benefits: It's simpler. It doesn't need to be a method.

Reply:
0762b0dfb072fc938b2d6d11445ae45b5c315ede